helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -237,6 +237,14 @@ spec:
     kind: Issuer
     name: {{ template "aws-load-balancer-controller.namePrefix" . }}-selfsigned-issuer
   secretName: {{ template "aws-load-balancer-controller.webhookCertSecret" . }}
+  {{- with .Values.certManager -}}
```

Comment:
This manifest will only be rendered when
```
{{- if not $.Values.enableCertManager }} 
```
it conflicts with your statement in README. Please clarify

Reply:
@oliviassss current behaviour is that If `enableCertManager` is true, `duration` and `renewBefore` fields are not specified in the Certificate resource and the implicit behaviour of the certmanager controller is that it will issue for 90d and renew after 60d.

This change allows to override this behaviour by  explicitly mentioning `duration` and `renewBefore` fields optionally if `enableCertManager` is true